pkg/virt-api/rest/definitions.go

```diff
@@ -594,7 +595,7 @@ func ClusterResourcePath(gvr schema.GroupVersionResource) string {
 
 func SubResourcePath(subResource string) string {
 	if !strings.HasPrefix(subResource, "/") {
-		return "/" + subResource
+		return filepath.Join("/", subResource)
```

Comment:
Not sure if `filepath.Join` is right here, this sounds like a URL path?

(Not sure what would happen here on Windows, besides that I don't believe it will ever run there, but you never know)

Reply:
Done